python/ray/train/v2/tests/test_worker_group.py

```diff
@@ -463,6 +467,8 @@ def after_worker_group_poll_status(self, worker_group_status):
     assert hooks.training_start_hook_called
     wg.poll_status()
     assert hooks.poll_status_hook_called
+    wg.abort()
+    assert hooks.abort_hook_called
```

Comment:
Let's add a unit test for `abort`

Reply:
moved to a separate test - agreed this makes more sense since we wouldn't call shutdown and abort in the same run.